packages/editor/src/components/properties/ParticleEmitterNodeEditor.tsx

```diff
@@ -172,8 +50,8 @@ export const ParticleEmitterNodeEditor = (props: ParticleEmitterNodeEditorProps)
         smallStep={0.01}
         mediumStep={0.1}
         largeStep={1}
-        value={props.node.ageRandomness}
-        onChange={onChangeAgeRandomness}
+        value={particleComponent.ageRandomness}
+        onChange={updateProperty(ParticleEmitterComponent, 'ageRandomness')}
```

Comment:
we can define a const for the same props being passed again and again, it would reduce the file length and increase code readability.

const StepProps = {
smallStep: 0.01,
mediumStep: 0.1,
largeStep: 1,
....
}

and can be used as

 <NumericInputGroup
    {...StepProps}
    value={particleComponent.ageRandomness}
    onChange={updateProperty(ParticleEmitterComponent, 'ageRandomness')}
  />